<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="new_connect_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: white;
	border: 1px solid lightgray;
	border-radius:5px;
}</string>
   </property>
   <property name="text">
    <string>+ New Connection</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="connect_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size:16px;
}</string>
   </property>
   <property name="text">
    <string>Connections :</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>531</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView{
	border: 1px solid lightgray;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size:16px;
}</string>
   </property>
   <property name="text">
    <string>Table :</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="connect_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>211</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTreeWidget{
	border: 1px solid lightgray;
}

QTreeWidget::item{
	height:25px;
	font-size:14px;
}
QHeaderView::section{
	height:25px;
	font-size:14px;
}
QTreeWidget::item:selected
{
    color:black;
	background-color:lightgray;
}</string>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>530</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: white;
	border: 1px solid lightgray;
	border-radius: 5px;
}</string>
   </property>
   <property name="text">
    <string>Add Row</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_btn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>530</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: white;
	border: 1px solid lightgray;
	border-radius:5px;
}</string>
   </property>
   <property name="text">
    <string>Delete Row</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
